2020/7xxx/CVE-2020-7036.json

```diff
@@ -1,18 +1,93 @@
 {
```

Comment:
('Expecting property name enclosed in double quotes: line 21 column 1 (char 625)',)